custodia/secrets.py

```diff
@@ -257,7 +257,12 @@ def _int_get_key(self, trail, request, response):
         key = self._db_key(trail)
         try:
             output = self.root.store.get(key)
-            if output is None:
+            # NOTE(rmascena): If the response value is empty, the
+            # row is a subcontainer, so only returns if it is
+            # requested properly.
+            if not output or output is None:
```

Comment:
Also I do not think we need a comment here, just mention this stuff in the commit message.